SU-HKKU/cardinal_service_profile
# CSD-7 Added related opportunities list at the bottom of opportunities pages

# READY FOR REVIEW

# Summary
- Added a view to the bottom of opportunity pages to display related items as tagged by opportunity type and dimension.

# Need Review By (Date)
- 5/26

# Urgency
- medium

# Steps to Test
1. `composer require SU-HKKU/cardinal_service_profile:dev-CSD-7-related-opps`
1. `blt drupal:sync --site=cardinalservice`
1. view any existing opportunity page
1. verify theres a list of 4 opportunities at the bottom of the page.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
